spec/models/journaled/change_writer_spec.rb

```diff
@@ -21,8 +21,8 @@
   end
 
   let(:model_class) do
-    double(
-      "SoldierClass",
+    class_double(
+      'Soldier',
```

Comment:
Would love a double check on the changes in this file.

I changed this to a `class_double` all the others to `instance_doubles` that operate on `ActiveRecord::Base`

This seemed a bit easier than standing up an actual table for these models to test against, but I could do that too!

Reply:
how does class_double with the string Soldier provide value as a verifying double when Soldier isn't actually a class?

Reply:
It does not at all, the docs state it falls back to a `double` is passed something it can't load.

So I think the best move here might be to just stand up a table, we already have pg in the test suite so seems cheap enough to at least create the table and give us a real AR class to use

Reply:
Pushed changes for this that stands up the Soldiers table in the dummy app, and reworks some of the mocking here!